index.html

```diff
@@ -316,8 +320,64 @@ <h3>
         </div>
     </section>
 
-    <script src="menu.js"></script>
+    </div>
+    <section class="popup_window">
+        <div class="popup">
+            <div class="mob_snap">
+                <img src="logo/mobile snap.png" alt="Mobile snapshoot">
+            </div>
+            <div class="desktop_snap">
+                <img src="logo/Desktop snap.png" alt="Desktop snapshoot">
+            </div>
+            <div class="popup-page">
+            <div class="popup_heading">
+                <h3>Keeping track of hundreds of components </h3>
+            </div>
+            <div class="popup_skills">
+                <ul class="popup_skills_mob">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <ul class="popup_skills_desktop">
+                    <li>Codekit</li>
+                    <li>GitHub</li>
+                    <li>JavaScript</li>
+                    <li>Bootstrap</li>
+                    <li>Terminal</li>
+                    <li>Codepen</li>
+                </ul>
+            </div>
+            <div class="popup_description_Mob">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
+
+                 <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
+
+            </div>
+            <div class="popup_description_Desktop">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+            </div>
+            </div>
+            <div class="popup_btn">
+                <button>
+                    <a href="">See live</a>
+                    <img src="logo/Icon - Export.svg" alt="live icon">
+                </button>
+                <button>
+                    <a href="">See source</a>
+                    <img src="logo/Frame.png" alt="source frame">
+                </button>
+            </div>
+            <span class="close">&times;</span>
+        </div>
```

Comment:
- [ ] Please 🙏🏻 remove the `popUp` model window static code from your **HTML** file. Rendering the modal window dynamically using JavaScript allows for a more flexible and efficient approach compared to statically placing it inside the HTML. By dynamically generating the modal window through JavaScript, we can manipulate its content, appearance, and behaviour programmatically. This enables dynamic updates, customization, and efficient reuse of the modal window across different parts of the application. It also provides greater control over the timing and conditions for displaying the modal, enhancing the user experience and allowing for more interactive and responsive interfaces.

Reply:
Kindly because of time can it be optional suggestion for now.
I will implement it.

Reply:
@Cheptoo21 No it can't be optional